torch/quantization/fx/qconfig_utils.py

```diff
@@ -13,6 +13,21 @@
 QConfigAny = Union[torch.quantization.QConfig,
                    torch.quantization.QConfigDynamic, None]
 
+def add_device_to_obs_ctr_in_qconfig(qconfig_dict, device='cpu'):
+    if '' in qconfig_dict.keys() and isinstance(qconfig_dict[''], torch.quantization.QConfig):
```

Comment:
Why is this only specific to keys that are `''`. For other entries in the qconfig dict I think we need to have similar init?

Reply:
this is fixed in the most recent version. Although I'm not sure if there's a better way to handle other types, it seems like other qconfig types don't necessarily live in qconfig_dict["<key_name>"].